### ISSUE TITLE
ProtonMail supports 2FA

### ISSUE THREAD

**Issue by andryou** [2factorauth/twofactorauth#2278]
https://protonmail.com/support/knowledge-base/two-factor-authentication/

**Comment by Jawshy** [comment]
@andryou, this site has already been updated via https://github.com/2factorauth/twofactorauth/commit/d22e5f7b9a00d21a5d637cdc723c273fa823e789. The changes will go live once `master` is merged into `gh-pages`. Please search before submitting a new issue.

**Comment by mxxcon** [comment]
ProtonMail should really stop spamming this project. We had 8 issues raised about this. 👎

**Comment by andryou** [comment]
Thanks, I created this issue because I've seen lots of tweets regarding 2FA and ProtonMail: https://twitter.com/search?f=tweets&q=%40ProtonMail%20Security%20is%20important%2C%20We%27d%20like%20it%20if%20you%20supported%20two%20factor%20auth&src=typd

**Comment by mxxcon** [comment]
We've never had so many issues about a single site for exactly the same topic. 😕

**Comment by andryou** [comment]
Please note that I'm not affiliated with ProtonMail. Thanks.
